modules/EvseV2G/iso_server.cpp

```diff
@@ -174,18 +174,19 @@ static void convertIso1ToXmldsigSignedInfoType(struct xmldsigSignedInfoType* xml
     xmld_sig_signed_info->SignatureMethod.ANY_isUsed = 0;
 }
 
-static bool load_contract_root_cert(mbedtls_x509_crt *contract_root_crt, const char* V2G_file_path, const char* MO_file_path) {
+static bool load_contract_root_cert(mbedtls_x509_crt* contract_root_crt, const char* V2G_file_path,
+                                    const char* MO_file_path) {
     int rv = 0;
 
     if (((rv = mbedtls_x509_crt_parse_file(contract_root_crt, MO_file_path)) != 0) &&
         ((rv = mbedtls_x509_crt_parse_file(contract_root_crt, V2G_file_path)) != 0)) {
```

Comment:
This isn't part of this PR, but still could be flagged, because this checks if two loads fail at the same time, and just then logs an error. If the first load fails and the second succeeds, it returns true.  Don't think this is correct

Reply:
Fix moved to seperate PR

Reply:
The if is correct. The contract leaf certificate can be based on the MO or on the V2G root certificate. Only if both root certificates can not be found, the leaf and sub ca certificates can not be verified.